applications/tari_console_wallet/src/grpc/wallet_grpc_server.rs

```diff
@@ -807,18 +792,22 @@ impl wallet_server::Wallet for WalletGrpcServer {
             "Committee definition transaction: {:?}", transaction
         );
 
+        let committee_size = match side_chain_features.constitution {
+            Some(constitution) => constitution.validator_committee.members().len(),
+            None => 0,
+        };
```

Comment:
Nit: This can be repalced by 
```suggestion
        let committee_size = side_chain_features.constitution.and_then(|con| con.validator_committee.members().len()).unwrap_or(0);
```